htmd/protocols/Amber_protocols/Production.in

```diff
@@ -0,0 +1,10 @@
+ test namelist
+ &cntrl
+   imin=0, ntx=5, irest=1, 
+   ntc=2, ntf=2, 
+   nstlim=12500000, ntt=3, gamma_ln=1.0,
+   temp0=300.0,
+   ntpr=5000, ntwr=500000, ntwx=5000,
+   dt=0.004, ig=-1, 
+   ntb=2, ntp=1, cut=8.0, barostat=2, ioutfm=1,ntxo=2,
+ /
```

Comment:
Hey, that is what we are doing now in the `htmd/acemd/amber.py` class. We are going through your comments too. Thank you for your quick feedback!